pages/agent/v3.md

```diff
@@ -165,6 +136,56 @@ The job API is unavailable on agents running versions of Windows before build 17
 >🛠 Experimental feature
 > To use the job API, set <code>experiment="job-api"</code> in your <a href="/docs/agent/v3/configuration#experiment"> agent configuration</a>.
 
+### Agent API
+
+Like the Job API experiment above, this exposes a (separate) local API for interacting with the agent process.
+The Agent API offers these endpoints:
+
+- `GET /api/leader/v0/ping` - Returns a JSON object with the current time (useful for testing the agent is alive).
+- `GET /api/leader/v0/lock?key=<key>` - Returns a JSON object containing the current state of a lock.
+- `PATCH /api/leader/v0/lock?key=<key>` - Accepts a JSON object with old and new states for a lock. The lock is then updated atomically, and a JSON object describing whether the operation proceeded is returned
+
+The API is exposed via a Unix Domain Socket. Unlike the `job-api`, the path to the socket is not available via a environment variable - rather, there is a single (configurable) path on the system.
+
+>🛠 Experimental feature
+> To use the agent API, set <code>experiment="agent-api"</code> in your <a href="/docs/agent/v3/configuration#experiment"> agent configuration</a>.
+
+
+## Promoted experiments
+
+We've recently promoted some experiments into full features.
+
+### Git mirrors
+
+Maintain a single bare git mirror for each repository on a host that is shared amongst multiple agents and pipelines. Checkouts reference the git mirror using `git clone --reference`, as do submodules.
+
+>🎓 Promoted experiment
+>
+> Git mirrors became a default feature in Agent v3.47.0. It can be used by setting the `--git-mirrors-path` flag.
```

Comment:
i'm not sure `default` is necessarily the right word here; it's not on by default, it's just not experimental anymore. maybe 
```suggestion
> Git mirrors became a full feature in Agent v3.47.0. It can be used by setting the `--git-mirrors-path` flag.
```

is good enough?

Reply:
Good point! Thanks